bluesky/callbacks/zmq.py

```diff
@@ -43,8 +46,13 @@ def __init__(self, address, *, RE=None, zmq=None):
         self.hostname = socket.gethostname()
         self.pid = os.getpid()
         url = "tcp://%s:%d" % self.address
-        self._prefix = b'%s %d %d ' % (self.hostname.encode(),
-                                       self.pid, id(RE))
+        if prefix is not None:
+            self._prefix = b'%s %s %d %d ' % (prefix, self.hostname.encode(),
+                                              self.pid, id(RE))
+        else:
+            self._prefix = b'%s %s %d %d ' % (self.hostname.encode(), self.pid,
```

Comment:
You don't need two `%s` in this case since you have 3 elements to format.

Reply:
thanks